public/pages/CreateDetector/components/ConfigureFieldMapping/components/RequiredFieldMapping/FieldNameSelector.tsx

```diff
@@ -15,43 +14,54 @@ interface SIEMFieldNameProps {
 }
 
 interface SIEMFieldNameState {
-  selectedOption?: string;
+  selectedOptions: EuiComboBoxOptionOption<string>[];
   errorMessage?: string;
 }
 
 export default class FieldNameSelector extends Component<SIEMFieldNameProps, SIEMFieldNameState> {
   constructor(props: SIEMFieldNameProps) {
     super(props);
+
+    let selectedOptions = [] as EuiComboBoxOptionOption<string>[];
```

Comment:
Rewrite as `let selectedOptions: EuiComboBoxOptionOption<string>[] = [];`?

Reply:
That makes sense, will do that.